code/game/objects/items/toys/crayons.dm

```diff
@@ -87,24 +87,24 @@
 	if(drawtype == "letter")
 		drawtype = tgui_input_list(user, "Choose the letter.", "Crayon scribbles", list("a","b","c","d","e","f","g","h","i","j","k","l","m","n","o","p","q","r","s","t","u","v","w","x","y","z"))
 
-	user.visible_message("[user] starts drawing something on \the [target.name]")
+	user.visible_message(span_notice("[user] starts drawing something on \the [target.name]."))
 	if(!instant && !do_after(user, 5 SECONDS, NONE, target, BUSY_ICON_GENERIC))
 		return
 
 	new /obj/effect/decal/cleanable/crayon(target, colour, shadeColour, drawtype)
 	uses--
 	if(uses <= 0)
-		balloon_alert_to_viewers("used up the crayon")
+		to_chat(user, span_notice("\The [src] breaks apart in your hand."))
 		qdel(src)
 
 /obj/item/toy/crayon/attack(mob/living/M, mob/living/user)
 	if(M != user)
 		return ..()
 
-	balloon_alert_to_viewers("takes a bite of \the [src] and swallows it")
+	user.visible_message(span_notice("[user] takes a bite of \the [src] and swallows it."))
```

Comment:
definitely good to remove the balloon spam here, but you seem to have missed it in /obj/item/reagent_containers/food/snacks/attack() where the main food balloon spam happens